libs/components/packages/src/schematics/rules/convert-definition-list-to-description-list/convert-definition-list-to-description-list.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+import { firstValueFrom } from 'rxjs';
+
+import { createTestApp } from '../../testing/scaffold';
+
+import { convertDefinitionListToDescriptionList } from './convert-definition-list-to-description-list';
+
+describe('Convert Definition List to Description List', () => {
+  const runner = new SchematicTestRunner(
+    'schematics',
+    require.resolve('../../../../collection.json'),
+  );
+
+  it('should convert definition list to description list', async () => {
```

Comment:
Would be good to have a few tests that show how the schematic handles:
- `*ngFor`, `@for`, `*ngIf`, etc.
- template reference variables
- other, unrecognized attributes: `data-*`, `class`, etc.